### PR TITLE
Changes to dockerfile to pass container check on preflight

### PR DESCRIPTION
**Pull request by yashoza19**
fixes https://github.com/redhat-openshift-ecosystem/operator-certification-operator/issues/45.

### CODE REVIEW

**Comment by itroyano** [comment]
Will need to find a way to automate incrementing/setting `release` and `version` during a build.
both in Makefile and in the Github Actions.
pending task for me later :)

**Comment by itroyano** [comment]
How much size does the image built from `ubi-minimal` have?

Current Operator images are a few dozen MB as seen in https://quay.io/repository/opdev/operator-certification-operator?tab=tags

Also, do we want to use `latest` or a specific tag / sha ?

**Comment by itroyano** [comment]
Can we copy the `LICENSE` from the source root folder into /licenses instead of adding a new file to the source?

`#copy license`
`COPY LICENSE /licenses/LICENSE`

**Comment by yashoza19** [comment]
> Can we copy the `LICENSE` from the source root folder into /licenses instead of adding a new file to the source?
> 
> `#copy license` `COPY LICENSE /licenses/LICENSE`

@itroyano Yes, that makes sense. I could revert back the License directory and make changes to the dockerfile instead. Thanks.

**Comment by yashoza19** [comment]
> How much size does the image built from `ubi-minimal` have?
> 
> Current Operator images are a few dozen MB as seen in https://quay.io/repository/opdev/operator-certification-operator?tab=tags

Size of the controller image built with ubi-minimal is around 60 MB. (https://quay.io/repository/yoza/operator-certification-operator)

File: Dockerfile [hunk 1]
```diff
@@ -18,9 +18,22 @@ RUN go mod download
...
+    version="0.0.0" \
+    name="operator-certification-operator" \
+    License="Apache-2.0" \
+    io.k8s.display-name="operator-certification-operator bundle" \
+    io.k8s.description="bundle for the operator-certification-operator" \
+    summary="This is the bundle for the operator-certification-operator" \
+    maintainer="opdev" \
+    vendor="Red Hat" \
+    release="${release_tag}" \
+    description="A Kubernetes operator to provision the OpenShift Operator Pipeline."
```
Comment (samira-barouti) [inline Dockerfile:34]: The operator does not solely provision the pipeline. I believe along the line `A Kubernetes operator to provision resources for the operator certification` makes more sense. WDYT?

Comment (yashoza19) [reply Dockerfile:34]: Yes, that makes sense. I copied the description from the repository's README. Will update the description to the above suggested. Thanks.

**Review by acornett21 — APPROVED** [review]

**Review by jomkz — APPROVED** [review]

**Pull request merged**